src/reports/components/fast-pass-report.tsx

```diff
@@ -40,48 +40,54 @@ export type FastPassReportProps = Omit<
+    return (
+        <>
+            <WebReportHead />
+            <BodySection>
+                <AutomatedChecksHeaderSection {...props} />
+                <ContentContainer>
+                    <FastPassTitleSection />
+                    <DetailsSection {...props} />
 
-                <ResultsContainer {...props}>
-                    <FastPassResultsTitleSection title="Automated checks" />
-                    <FastPassReportAutomatedChecksResults {...props} />
-                    <FastPassResultsTitleSection key={4} title="Tab stops" />
-                    <TabStopsFailedInstanceSection
-                        key={5}
-                        deps={{
-                            tabStopRequirementActionMessageCreator: undefined,
-                            tabStopsTestViewController: undefined,
-                            ...props.deps,
-                        }}
-                        tabStopRequirementState={props.results.tabStops}
-                        alwaysRenderSection={true}
-                    />
-                    <TabStopsChecksSectionWrapper
-                        key={6}
-                        checksSection={IncompleteChecksSection}
-                        tabStops={props.results.tabStops}
-                        {...props}
-                    />
-                    <TabStopsChecksSectionWrapper
-                        key={7}
-                        checksSection={PassedChecksSection}
-                        tabStops={props.results.tabStops}
-                        {...props}
-                    />
-                </ResultsContainer>
-            </ContentContainer>
-            <ReportFooter>
-                <FooterText {...props} />
-            </ReportFooter>
-        </BodySection>
-    </>
-));
+                    <FastPassReportSummary {...props} />
+
+                    <ResultsContainer {...props}>
+                        <FastPassResultsTitleSection title="Automated checks" />
+                        <FastPassReportAutomatedChecksResults {...props} />
+                        <FastPassResultsTitleSection key={4} title="Tab stops" />
+                        <TabStopsFailedInstanceSection
+                            key={5}
+                            deps={{
+                                tabStopRequirementActionMessageCreator: undefined,
+                                tabStopsTestViewController: undefined,
+                                ...props.deps,
+                            }}
+                            tabStopRequirementState={props.results.tabStops}
+                            alwaysRenderSection={true}
+                        />
+                        <TabStopsChecksSectionWrapper
+                            key={6}
+                            checksSection={IncompleteChecksSection}
+                            tabStops={props.results.tabStops}
+                            {...props}
+                            testKey={tabStopsTestKey}
```

Comment:
One thought: since `testKey` is only set to tab-stops here, it seems reasonable that `TabStopsChecksSectionWrapper` could take responsibility for setting it and `FastPassReport` could remain unchanged. Just an idea; totally optional!

Reply:
This is a good call, I made the change! :)